core/test.ts

```diff
@@ -168,13 +168,12 @@ class RefReplacingContext implements ITableContext {
     return "";
   }
 
-  public isIncremental() {
-    return false;
+  public when(cond: boolean, trueCase: string, falseCase?: string) {
+    return "";
```

Comment:
this should have the same behaviour as `table.ts`

Reply:
Ah ok